gatsby-config.js

```diff
@@ -130,14 +130,25 @@ module.exports = {
               return allMarkdownRemark.edges
                 .filter((edge) => !edge.node.fields.slug.includes('/pages/'))
                 .map((edge) => {
-                  return Object.assign({}, edge.node.frontmatter, {
+                  const blogPostMeta = edge.node.frontmatter;
+                  const imageUrl =
+                    BASE_URL +
+                    (blogPostMeta.shareImage.childImageSharp.fixed.src ||
+                      DEFAULT_META_IMAGE_URL_PATH);
+
+                  return {
+                    title: blogPostMeta.title,
+                    site_url: site.siteMetadata.siteUrl,
+                    date: blogPostMeta.date,
                     description: edge.node.excerpt,
-                    date: edge.node.frontmatter.date,
-                    url: site.siteMetadata.siteUrl + edge.node.frontmatter.path,
-                    guid:
-                      site.siteMetadata.siteUrl + edge.node.frontmatter.path,
-                    custom_elements: [{ 'content:encoded': edge.node.html }],
-                  });
```

Comment:
Asking because it looks similar: Did something change here or is this only due to the introduction of the `blogPostMeta` refactoring ?

Reply:
Only because of the variable name change.